pkg/remote/message.go

```diff
@@ -183,7 +183,7 @@ func (m *message) ServiceInfo() *serviceinfo.ServiceInfo {
 }
 
 func (m *message) SpecifyServiceInfo(svcName, methodName string) (*serviceinfo.ServiceInfo, error) {
-	// for non-multi-service including generic server scenario
+	// for single service scenario
```

Comment:
After supporting generic multi services, the logic here needs to be adjusted. Mark a TODO here.

Reply:
@jayantxie I don't think the logic here needs to be adjusted even when supporting generic multi-service since now generic has the real service name in svcInfo.

Reply:
generic function name is only `$GenericMethod`, so there's no map key to find the real service.